@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -850,7 +856,7 @@ export class TextElementsRenderer {
 
         // Do the actual sort based on sortPriority
         textElements.sort((a: TextElement, b: TextElement) => {
-            return b.sortPriority! - a.sortPriority!;
+            return a.sortPriority! - b.sortPriority!;
```

Comment:
Noticed that it preferred the labels in the distance. I think that this works better ;)

Reply:
According to https://www.geeksforgeeks.org/javascript-array-sort/, this would mean that for example, if A has sort priority 10, and b 5, that b would come before a. Which means that lower priority items are tried first, and would have a higher chance of being visible, seems counter intuitive. That is ok, if a lower numeric priority means a higher actual priority, (but this should be documented on the sortPriority property)

Reply:
Reversed the change.